platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -1,150 +1,161 @@
 ## Introduction
 
-Inspired by the cross-functional cooperation promised by DevOps, platform
-engineering has begun to emerge in enterprises as an explicit form of that
-cooperation. Platforms curate and present foundational capabilities, frameworks
-and experiences to facilitate and accelerate the work of internal customers such
-as application developers, data scientists and information workers. Particularly
-in the field of cloud computing, platforms have helped enterprises realize
-values long promised by cloud like fast product releases, portability across
-infrastructures, more secure and resilient products, and greater developer
+Inspired by the cross-functional cooperation promised by DevOps, platform 
+engineering has begun to emerge in enterprises as an explicit form of that 
+cooperation. Platforms curate and present foundational capabilities, frameworks 
+and experiences to facilitate and accelerate the work of internal customers such 
+as application developers, data scientists and information workers. Particularly 
+in the field of cloud computing, platforms have helped enterprises realize 
+values long promised by cloud like fast product releases, portability across 
+infrastructures, more secure and resilient products, and greater developer 
 productivity.
 
-This paper intends to support enterprise leaders, enterprise architects and
-platform team leaders to advocate for, investigate and plan internal platforms
-for cloud computing.  We believe platforms significantly impact enterprises'
-actual value streams, but only indirectly, so leadership consensus and support
-is vital to the long-term sustainability and success of platform teams. In this
-paper we'll enable that support by discussing what the value of platforms is, how
+This paper intends to support enterprise leaders, enterprise architects and 
+platform team leaders to advocate for, investigate and plan internal platforms 
+for cloud computing.  We believe platforms significantly impact enterprises' 
+actual value streams, but only indirectly, so leadership consensus and support 
+is vital to the long-term sustainability and success of platform teams. In this 
+paper we'll enable that support by discussing what the value of platforms is, how 
 to measure it, and how to implement platform teams that maximize it.
 
 ## Table of Contents
 
+1. Why have internal platforms gained popularity
```

Comment:
I think this is a good point. I didn't realise how big of a shift in language this was until you pointed it out. I do think this would be a valuable discussion at the next group call if possible 🤔 

My understanding is that we are talking about internal here and it would help us to be clear about that. Of course “platform” can be internal, external or anything else. But we are looking at how to enable and optimise the delivery of business value.

I took a stab at a clarifying sentence [here](https://github.com/cncf/tag-app-delivery/pull/294/files#r1095899200) if it helps 🤞

Reply:
Interesting discussion. Do you think an external company can‘t provide such a platform? (On prem or hosted as-a-service)?

Reply:
Thanks for asking Johannes!

Just to say at the start, I definitely agree that an external company / tool can provide a part (often a large part) of a platform either on prem or hosted.

I think that was has been revealed by my statement is that we may need to discuss and align on what a platform is (timely given our white paper topic 😅 ). I have had people reach out and ask about tools like Heroku (or the many newer offerings in the space) as an example of how a tool can be a platform for many organisations. IMO tools like this are not a complete platform for a company of scale.

Let me break down that statement as it seems to be the controversial one and I would love to get to an even better and shared opinion:

*offering like Heroku (or the many new versions) is not a complete platform* : This refers to offerings that host applications. They often support needs like building, deploying, and scaling an application. They _might_ support something like basic data storage, but more likely they offer integrations into other hosted solutions for that. This could be internal to their offerings (think AppEngine and CloudSQL from GCP). These are an amazing quick start for small orgs that are mostly concerned about go to market. They are often not enough for organisations that need more control over other pieces (e.g. adding queuing, caching, or other things).

*a company of scale*: This is going to be very subjective, but I think that the important thing is to confirm if my assumption about this white paper is correct. I believe we are targeting "enterprises" which I would assume have at least hundreds of engineers. IME size is not the only indication of complexity, but I rarely see an enterprise level company that does not have needs around PCI DSS / SOC2 / ISO-270001 style auditing, reliability / disaster recovery requirements that depend on redundancy, and a number of environments / testing support that requires additional tooling and support.

I suspect these assumptions / definitions will evolve, but using them as a starting point, I would say in the case of an enterprise, it is highly unlikely that any external tool / company can be the companies main interface for engineers, even if it can be the underlying tool for large parts of their problem space. One counter example to this may be a UI / API specific tool (e.g. Backstage) which can act as an un-opinionated framework to provide a single of entry for any number of tools behind the scene.